### ISSUE TITLE
The additional content, which is being displayed on mouse hover, is disappearing while moving focus on it, at Navigation screen.

### ISSUE THREAD

**Issue by rgarg24** [percussion/percussioncms#868]
**Describe the bug**
The additional content, which is being displayed on mouse hover, is disappearing while moving focus on it, at Navigation screen.

**To Reproduce**
1. Go to Navigation screen and select any page for a site and navigate it.
2. Mouse hover on the 'Delete Site' button. It will show "Delete Site" label.
3. Now try move the pointer over the "Delete Site", the label disappears.

**Expected behavior**
User should be able to move pointer over the additional content (tool tip, label etc.), and this additional content should not disappear, while moving focus on it.

**Additional context**
Navigate all the options available for similar issue and fix.

**Comment by rgarg24** [comment]
There is no additional content but title. So, closing this bug.
